main.go

```diff
@@ -25,11 +25,14 @@ var (
 )
 
 const (
-	stateDesc      string = "Optional; the state JSON output by 'terraform show -json', read from stdin if omitted"
+	stateDesc      string = "Optional; the state JSON output by 'terraform show -json'. Read from stdin if omitted"
 	stateFileDesc  string = "Optional; the path to a local file containing 'terraform show -json' output"
-	headingDesc    string = "Optional; the heading text for use in the printed markdown"
+	headingDesc    string = "Optional; the heading text for use in the printed output."
+	outputDesc     string = "Optional; the output format. Supported values: md, html."
```

Comment:
```suggestion
	outputDesc     string = "Optional; the output format. Supported values: md, html. Defaults to md if omitted"
```

Reply:
Thanks, though I _believe_ this isn't necessary because the `flags` package adds it on our behalf:

```
terraputs -h
Usage of terraputs:
  -heading string
        Optional; the heading text for use in the printed output. (default "Outputs")
  -output string
        Optional; the output format. Supported values: md, html. (default "md")
  -state string
        Optional; the state JSON output by 'terraform show -json'. Read from stdin if omitted
  -state-file string
        Optional; the path to a local file containing 'terraform show -json' output
```

Reply:
Ah, cool.